### DISCUSSION TITLE
Aligned size dock icons

### DISCUSSION THREAD

**Discussion by ghost** [linuxdeepin/developer-center#3182] [Features Request & Ideas | 特性请求 & 头脑风暴]
I have a suggestion, some dock icons especially default Deepin apps trend to have bigger icons and less aligned compared to third party installed apps. As an example, Deepin Browser icon looks bigger than Microsoft Edge. Not speaking Discord icon looks the smallest of them all 
![Screenshot_dde-dock_20220526141017](https://user-images.githubusercontent.com/98664445/170550851-e309e29c-7566-4f0b-bac5-dde88366fec8.png)

**Comment by BLumia** [comment]
As a workaround, you can put your own icon under [somewhere in XDG_DATA_HOME](https://stackoverflow.com/questions/58060043/where-to-install-icon-for-a-user-specific-application-while-adhering-to-freedesk) or switch an icon theme to use other icons.

**Comment by ghost** [comment]
I like bloom, dark vs default one is basically same with those size icons. As a future perspective I pointed the suggestions in the next versions to be changed, maybe v23 that is coming 😎
